lib/jekyll-feed/meta-tag.rb

```diff
@@ -17,21 +33,50 @@ def config
       @config ||= @context.registers[:site].config
     end
 
-    def attributes
+    def generator
+      @generator ||= @context.registers[:site].generators.select { |it| it.is_a? JekyllFeed::Generator }.first # rubocop:disable Metrics/LineLength
```

Comment:
Couple of things to change here:
  - RuboCop disabling comments should be placed above the method definition. A disabled cop should be enabled likewise, with the comment placed right after the filtered method(s).
  - Prefer using `Array#find` instead of chaining `Array#select#first`.
  - Block parameters should be immediately comprehensible. Even though it makes very little difference, a developer reading this code will spend a few seconds thinking if `it` means `"it"` or a shorthand for `"item"`.. :wink:
I'd prefer `generator` itself or `gen` for short.

```ruby
    def generator
      @generator ||= @context.registers[:site].generators.find do |generator|
        generator.is_a? JekyllFeed::Generator
      end
    end
```
```ruby
    def generator
      @generator ||= @context.registers[:site].generators.find { |gen| gen.is_a? JekyllFeed::Generator }
    end
```